spec/spec_helper.rb

```diff
@@ -8,6 +8,9 @@
   coverage_dir "public/coverage"
 end
 
+ENV["VELUM_SALT_HOST"] ||= "127.0.0.1"
```

Comment:
Why have you done this ? Is this relevant to this PR ? 😕

Reply:
Leftover! Thanks!

Reply:
Forgot to remove this?